support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/CassandraTicketRegistryDao.java

```diff
@@ -0,0 +1,29 @@
+package org.apereo.cas.ticket.registry;
+
+import org.apereo.cas.ticket.TicketGrantingTicket;
+
+import java.util.stream.Stream;
+
+/**
+ * @author David Rodriguez
+ *
+ * @since 5.1.0
+ */
+public interface CassandraTicketRegistryDao {
```

Comment:
Cool. I think there is opportunity to remove either the parent interface entirely here, or maybe that specific method by stuffing it some place else, but that can obviously happen later.